assets/src/blocks/Submenu/getHeadingsFromBlocks.js

```diff
@@ -4,81 +4,81 @@ import { unescape } from '../../functions/unescape';
 // blocks this should be sufficient.
-const stripTags = str => str.replace(/(<([^>]+)>)/ig, '');
+const stripTags = ( str ) => str.replace( /(<([^>]+)>)/ig, '' );
 
-export const getHeadingsFromBlocks = (blocks, selectedLevels) => {
+export const getHeadingsFromBlocks = ( blocks, selectedLevels ) => {
   const headings = [];
-  blocks.forEach(block => {
-    if (block.name === 'core/heading') {
+  blocks.forEach( ( block ) => {
+    if ( block.name === 'core/heading' ) {
       const blockLevel = block.attributes.level;
 
-      const levelConfig = selectedLevels.find(selected => selected.heading === blockLevel);
+      const levelConfig = selectedLevels.find( ( selected ) => selected.heading === blockLevel );
 
-      if (!levelConfig) {
+      if ( ! levelConfig ) {
         return;
       }
 
-      const anchor = block.attributes.anchor || generateAnchor(block.attributes.content, headings.map(h => h.anchor));
+      const anchor = block.attributes.anchor || generateAnchor( block.attributes.content, headings.map( ( h ) => h.anchor ) );
 
-      headings.push({
+      headings.push( {
         level: blockLevel,
         // The content of RichText elements will always come out escaped. This is problematic as those will be displayed
         // literally when we render them. It seems safe to unescape here as the value will not be used without escaping.
-        content: unescape(stripTags(block.attributes.content)),
+        content: unescape( stripTags( block.attributes.content ) ),
         anchor,
         style: levelConfig.style,
         shouldLink: levelConfig.link,
-      });
+      } );
 
       return;
     }
 
-    if (block.name === 'core/freeform') {
+    if ( block.name === 'core/freeform' ) {
       const parser = new DOMParser();
-      const selector = selectedLevels.map(({heading})=>`h${heading}`).join();
-      const doc = parser.parseFromString(block.attributes.content, 'text/html');
+      const selector = selectedLevels.map( ( { heading } ) => `h${ heading }` ).join();
+      const doc = parser.parseFromString( block.attributes.content, 'text/html' );
 
-      const classicHeadings = doc.querySelectorAll(selector);
+      const classicHeadings = doc.querySelectorAll( selector );
 
-      headings.push(...[...classicHeadings].map(h => {
-        const blockLevel = parseInt(h.tagName.replace('H', ''));
-        const levelConfig = selectedLevels.find(selected => selected.heading === blockLevel);
+      headings.push( ...[ ...classicHeadings ].map( ( h ) => {
+        const blockLevel = parseInt( h.tagName.replace( 'H', '' ) );
+        const levelConfig = selectedLevels.find( ( selected ) => selected.heading === blockLevel );
 
-        const anchor = h.id || generateAnchor(h.innerText, headings.map(h => h.anchor));
+        const anchor = h.id || generateAnchor( h.innerText, headings.map( ( headder ) => headder.anchor ) );
```

Comment:
Small typo here, should be `header` (or maybe `heading` to be more consistent), not `headder`

Reply:
Done